src/utils/sentry-config.ts

```diff
@@ -13,4 +13,20 @@ export const sentryConfig: ModuleConfiguration = {
     'https://53da8fbcebeb48a6bf614a212629df6b@o787041.ingest.sentry.io/5799642',
   disabled: !isProd,
   lazy: true,
+  clientConfig: {
+    // Only allow errors that come from an actual openverse.engineering subdomain
+    allowUrls: [/^https?:\/\/(.*)\.openverse\.engineering/],
```

Comment:
Good thing this is configurable! Should we include the [staging URL](https://api-dev.openverse.engineering/) too?

Reply:
Yes! It should be much easier than trying to explicitly _exclude_ everything else (which is what I tried first before I realized there was a much easier way :sweat_smile: )